src/cf-forms/src/cf-forms.less

```diff
@@ -30,12 +30,27 @@
 // .disabled
 @input-disabled:                #aeb0b5; // $color-gray-light
 
+// Large target area variables
+@input-big-target-bg:           #d6d7d9; // $color-gray-lighter
+
+@input-big-target-bg__selected: #9bdaf1; // $color-primary-alt-light
+@input-border__selected:        #046b99; // $color-primary-alt-darkest
+
+@input-big-target-bg__disabled: #aeb0b5; // $color-gray-light
+
 // Sizing variables
 
 // .input__super
 @input__super-font-size:        18px;
 
 //
+// Import molecules and atoms
```

Comment:
Also I like this structure, should I use the same for the select in PR #350? (in a subsequent pr, that one has languished long enough)